cmd/utils.go

```diff
@@ -0,0 +1,46 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+// ProjectType ...
+type ProjectType string
+
+const (
+	// iOS
+	xcodeWorkspace ProjectType = "*.xcworkspace"
+	xcodeProject   ProjectType = "*.xcodeproj"
+)
+
+// Scans the root dir for the provided project files
+// If none of them in the root dir, then it will return an error
+func scanForProjectFiles(projectFiles []ProjectType) (string, error) {
+	root, err := os.Getwd()
+	if err != nil {
+		return "", err
+	}
+
+	for _, projectFile := range projectFiles {
+		pathPattern := filepath.Join(root, string(projectFile))
+		paths, err := filepath.Glob(pathPattern)
+		if err != nil {
+			return "", err
+		}
+
+		switch len(paths) {
+		case 0:
+			continue
```

Comment:
this case will happen anyway, so it is useless here(because you have a while switch block as the last lines in the for scope)

Reply:
if there is only 2 case then please use if